core/new-gui/src/app/workspace/types/execute-workflow.interface.ts

```diff
@@ -30,9 +30,21 @@ export interface LogicalPlan extends Readonly<{
 /**
  * The backend interface of the return object of a successful execution
  */
+
+ 
+
+export interface ResultObject extends Readonly<{
+  operator: string,
+  table: ReadonlyArray<object>,
+  chartType: string | undefined
+}> {
+
+}
 export interface SuccessExecutionResult extends Readonly<{
   code: 0,
-  result: ReadonlyArray<object>,
+  result: ReadonlyArray<ResultObject>,

```

Comment:
to echo an ealier review, just change the interface here instead of creating a new interface. Also make sure that the backend's return value is the same as the interface declared here.